artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnection.java

```diff
@@ -511,6 +511,45 @@ public final boolean isUsingProtocolHandling() {
       return true;
    }
 
+   @Override
+   public boolean isSameTarget(TransportConfiguration... configs) {
+      boolean yes = false;
```

Comment:
hehehe

Reply:
@gaohoward  can you rename it from yes? :)

call it result if you like :)

Reply:
alright. :)